apps/server/src/connect/userEndpoints.ts

```diff
@@ -10,11 +12,31 @@ export interface UserDeleteRequest {
   params: UserDeleteParameters
 }
 
+const createProviderValidator = () =>
```

Comment:
I think this is at least the 3rd time this has been used. We should put this function in a shared file somewhere

Reply:
Oh yeah I meant to do that before creating a PR. I'll fix this